cmadison/cmadison.py

```diff
@@ -130,14 +135,15 @@ def get_sources(self):
         :param filename: the file to read the source packages from.
         """
         lines = []
-        for line in gzip.open(self.fname):
-            # Empty line is the end of the source package stanza
-            if line.strip() == '':
-                src = Source.parse(''.join(lines))
-                lines = []
-                yield src
-            else:
-                lines.append(line)
+        if self.ready:
```

Comment:
Since python is heavily dependent on indentation, I tend to prefer code changes which reduce indentation as much as possible, ergo I prefer:

```python
if not self.ready:
    return
```